doc/networking.md

```diff
@@ -0,0 +1,105 @@
+This bridge network device shows up as `docker0` when running `ifconfig`.
+
+The remaining containers, including the Edge Hub, are placed on a [user-defined bridge network][2] (named `azure-iot-edge`).
+
+## Default Bridge Network (docker0)
+
+The Edge Agent is started by `iotedged` and is started without any explicit networking configuration.
+By default, the Moby runtime places all containers without a defined network configuration on the default bridge network (`bridge`).
+
+The Edge Agent requires outbound internet connectivity to IoT Hub to function properly.
+This means that a route from the default docker bridge subnet to the internet must exist, no firewall rules are setup to block traffic, and IP forwarding is enabled.
+All three of these conditions are met in a standard Docker installation.
+
+### Configuring `bridge`
+
+The Docker Engine creates and configures the default bridge network.
+Changing this configuration is done at the Docker Engine level, by modifying the `daemon.json` config file and restarting the Docker Engine.
+This file is normally located at `/etc/docker/daemon.json` on Linux.
+
+Reasons for changing this configuration include:
+* Modifying the IP block used for assigning IPs to containers
+* Changing the MTU
+* Changing the gateway address
+
+The usual reason for modifying this configuration is that some other subnet on the network clashes with the bridge network subnet.
+
+## `azure-iot-edge` Network
+
+Modules (containers), including the Edge Hub, are started by the Edge Agent and placed on a user-defined bridge network named `azure-iot-edge`.
+This network is created when the `iotedged` boots for the first time.
+
+The Edge Hub requires outbound internet connectivity to IoT Hub to function properly.
+This means that a route from the bridge subnet to the internet must exist and no firewall rules are setup to block traffic.
+
+### Configuring `azure-iot-edge`
+
+This network name is configured via the `moby_runtime` section of the `iotedged`'s config file (`/etc/iotedge/config.yaml):
+
+```yaml
+moby_runtime:
+  uri: "unix:///var/run/docker.sock"
+  network: "azure-iot-edge"
+```
+
+Any changes to the specific settings of this network must be made out of band, via the Docker Engine.
+Read the Docker [networking guide][4] for more information.
+
+This network can be configured before starting IoT Edge, as the `iotedged` does a "get or create" operation on the network when booting.
+Pre-configuring a network and updating the network in the `config.yaml` allows complete control over its settings.
+
+# Ports
+
+IoT Edge does not require any inbound ports to be open for proper operation.
+
+IoT Edge does require several ports to be open for outbound connectivity.
+The following table describes these requirements:
+
+|Protocol | Port | Inbound                         | Outbound  |
+|---------|------|---------------------------------|-----------|
+| MQTT    | 8883 | OPTIONAL (For gateway scenario) | OPEN*     | 
```

Comment:
Should we expand on what "OPTIONAL (For gateway scenario)" means after the table?  Would it make sense to add the gateway scenario to your diagram?

Reply:
yes agree, this could do better with an explanation

Reply:
I added a quick paragraph talking about the gateway scenario.